ADAL/src/public/ADAuthenticationContext.h

```diff
@@ -28,7 +28,9 @@
 @class ADTokenCacheItem;
 @class ADUserInformation;
 @class ADUserIdentifier;
+#if TARGET_OS_IPHONE
 @class UIViewController;
```

Comment:
I don't think that @class UIViewController there is even necessary. I think it can be removed instead, but let me know if I'm missing something :)

Reply:
I'm not even sure, but I'm guessing it's required as an early declaration since the .h file there doesn't pull in any of the UI frameworks.